### ISSUE TITLE
Could you consider adding a function so that it can simply load .xml files?

### ISSUE THREAD

**Issue by dzx-dzx** [danicuestasuarez/NMapGUI#27]

**Comment by JustOneMoreDog** [comment]
Came here just to ask this.  I absolutely love this GUI you have created. Thank you so much for it.  This would make the program perfect in my eyes.  Sometimes I have to run nmap on machine where I cant get a gui due to limitations so being able to export the scan as xml and then import it into your amazing program would be mouth watering.  Also worth nothing that it would help if we could move the nodes on the graph around.  Right now the graph is only a nicer looking version of the zenmap one but with no extra functionality.

**Comment by daniel-cues** [comment]
@picnicsecurity Yeah, I'd like to keep implementing stuff for the project, but I'm waiting until after my degree's final project presentation (yeah I know it's taking long, can't help the procrastination).
Also, you should be able to move the nodes around holding left click and they should show their open ports when hovering over them

I'm happy you guys enjoy the project and I'll keep working on it soon.

**Comment by JustOneMoreDog** [comment]
Take your time there is no rush!  You have provided an absolutely amazing tool to the community.  Good luck with your project!
